### ISSUE TITLE
Failing test `tests/plugins/maximize/aria`

### ISSUE THREAD

**Issue by gebsh** [ckeditor/ckeditor4#5194]
## Type of report

Failing test

## Provide detailed reproduction steps (if any)

1. Run production unit tests on a mobile device with iOS in Safari browser.

### Expected result

All tests pass.

### Actual result

The test `the label is not changed after maximizing the editor` fails with message:

```txt
Test has no asserts.
Expected: undefined (undefined)
Actual:   undefined (undefined)
```

## Other details

* Browser: Safari 15.4
* OS: iOS 15.4.1
* CKEditor version: 4.19.0
* Installed CKEditor plugins: maximize

**Comment by Comandeer** [comment]
These tests should be ignored on mobile as maximize does not work there. I'll prepare a fix for that.

**Comment by CKEditorBot** [comment]
Closed in https://github.com/ckeditor/ckeditor4/pull/5195
